### PR TITLE
Update affiliation for a project member 

### PR DESCRIPTION
**Pull request by Reshrahim**
Updating the affiliation for https://github.com/sk593

### CODE REVIEW

File: developers_affiliations7.txt [hunk 1]
```diff
@@ -15605,12 +15605,7 @@ sk-: sk-!users.noreply.github.com
 sk4zuzu: mopala!opennebula.io
 	Independent
 sk593: shruthikumar2401!gmail.com, sk593!users.noreply.github.com
-	Independent until 2020-05-01
```
Comment (sk593) [inline developers_affiliations7.txt:15608]: I don't recognize these companies or dates. Wondering if there was a mixup in users somewhere?

Comment (lukaszgryglicki) [reply developers_affiliations7.txt:15608]: Don't know, sorry, I'll update according to this PR.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by Reshrahim** [comment]
@lukaszgryglicki - Thanks for approving this. How long does it take for the [dashboard](https://radius.devstats.cncf.io/d/4/company-statistics-by-repository-group?orgId=1&var-period=w&var-metric=activity&var-repogroup_name=All&var-companies=All) to reflect it correctly ?

**Comment by lukaszgryglicki** [comment]
Up to 28 days.

**Pull request merged**